### ISSUE TITLE
Population Characteristics: Compare populations (add more charts in config file)

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/catalogue#747]
Add in the configuration file to show more characteristics.

Also, the feature are hidden and show be shown (to dev version)

**Comment by ribeiroct** [comment]
Active Patients, follow-up time(patient time ?) were already implemented.

Meanwhile i implemented the missing year of birth and start year charts.

The number of patients is missing. The data isnt on the samples of jerboa i looked at, maybe it can be calculated from the sommatory of "start year" 

I think the aggregator is returning bad values(at least it looks that way to me).

**Comment by bastiao** [comment]
@ribeiroct is it already fixed?

I checked the Active patients and it looks fine in the aggregator. Comparison with 3 databases at same time.

Also, one thing that I did not implement in the aggregator is the hook to re-aggregate when something change in one databases. we may need for a async processo to recalculate all things. 
Maybe we can think about a scalable matter, but at the moment, we need to finish it and starting pushing the integration with Achilles.
